config/prooph_service_bus.config.php

```diff
@@ -0,0 +1,67 @@
+ * shipped with this package. Please refer to src/Factory for the factories and
+ * register them in your Interop\Container\ContainerInterface of choice
+ * Then make this config available as service id `config` within your container
+ * (possibly merged into your application configuration)
+ */
+return [
+    //vendor key to avoid merge conflicts with other packages when merged into application config
+    'prooph' => [
+        //component key to avoid merge conflicts with other prooph components when merged into application config
+        'service_bus' => [
+            //This section will be used by Prooph\ServiceBus\Factory\CommandBusFactory
+            'command_bus' => [
+                //You can add a list of container service ids
+                //The factory will use these to get the plugins from the container
+                'plugins' => [],
+                'router' => [
+                    //Map of message routes where the message name being the key and the command handler being the value
+                    //To lazy-load command handlers you can provide service id to command handler instead.
+                    //In this case the handler is pulled from the container using the provided handler service id
+                    'routes' => [
+
+                    ],
+                    //Router defaults to Prooph\ServiceBus\Plugin\Router\CommandRouter
+                    //Comment out the next line to use the RegexRouter instead
+                    //'type' => \Prooph\ServiceBus\Plugin\Router\RegexRouter::class,
+                ]
+            ],
+            //This section will be used by Prooph\ServiceBus\Factory\EventBusFactory
+            'event_bus' => [
+                //You can add a list of container service ids
+                //The factory will use these to get the plugins from the container
+                'plugins' => [],
+                'router' => [
+                    //Map of message routes where the message name being the key and the value being a list of event listeners.
+                    //To lazy-load event listeners you can provide service ids instead.
+                    //In this case each listener is pulled from the container using the provided listener service id
+                    'routes' => [
+
+                    ],
+                    //Router defaults to Prooph\ServiceBus\Plugin\Router\EventRouter
+                    //Comment out the next line to use the RegexRouter instead
+                    //'type' => \Prooph\ServiceBus\Plugin\Router\RegexRouter::class,
+                ]
+            ],
+            //This section will be used by Prooph\ServiceBus\Factory\QueryBusFactory
+            'query_bus' => [
+                //You can add a list of container service ids
+                //The factory will use these to get the plugins from the container
+                'plugins' => []
+
+            ],
+        ]
+    ]
+];
```

Comment:
arrrgh, I checked the template, but this f** space is added by PHPStrom. In my defense I have to say that the PR is not ready yet ;). I'll fix it with my next commit. I think I have to investigate some time to set up php-cs-fixer ...